src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -59,26 +62,30 @@ public BookDto getBookById(@PathVariable @Positive Long id) {
         return bookService.findById(id);
     }
 
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
```

Comment:
```suggestion
    @PreAuthorize("hasRole('ADMIN')")
```
Spring will add this prefix by itself